cogs/admin.py

```diff
@@ -1,10 +1,4 @@
-import time
```

Comment:
There's a reason it is in this layout, flake8 error.
However I haven't added a setup.cfg file to ensure that it is noted
![image](https://user-images.githubusercontent.com/19587382/172145500-3531a9c3-08a2-47c1-a0bd-e44c812590e3.png)

Reply:
Done for all files, sorry hah